<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassWord</class>
 <widget class="QWidget" name="PassWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: none;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>341</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: rgb(63, 63, 63);}
QFrame{border:1px groove #191919;}
QFrame{border-radius:18px;}
QFrame{padding:2px 4px;}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="MOK_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>98</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 118, 118);

 border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
    border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MCancel_Button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>98</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{border:none;}
QFrame{color: rgb(255, 255, 255);}
</string>
    </property>
    <property name="text">
     <string>请输入原始密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PwdLineEdit_old">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 1px solid #447684;
    color:black;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="PwdLineEdit_new">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 1px solid #447684;
    color:black;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{border:none;}
QFrame{color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>请输入新密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{border:none;}
QFrame{color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>请再次输入新密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PwdLineEdit_newc">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 1px solid #447684;
    color:black;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>391</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: rgb(63, 63, 63);}
QFrame{border:1px groove #191919;}
QFrame{border-radius:18px;}
QFrame{padding:2px 4px;}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>136</y>
      <width>98</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 118, 118);

 border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
    border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{border:none;}
QFrame{color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>当前操作模式是操作员，
无权限进行保存和修改。</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>341</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{background-color: rgb(63, 63, 63);}
QFrame{border:1px groove #191919;}
QFrame{border-radius:18px;}
QFrame{padding:2px 4px;}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{border:none;}
QFrame{color: rgb(255, 255, 255);}
</string>
    </property>
    <property name="text">
     <string>请输入密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PwdLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">    border: 1px solid #447684;
    color:black;
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Make_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 118, 118);

 border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
    border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ok_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 118, 118);

 border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
    border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancle_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>41</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(118, 118, 118);

 border-style: outset;
 border-width: 1px;
border-radius: 5px;
 border-color: #191919;
 color:white;
padding: 1px;
    border: 1px solid #447684;
background-color: rgb(85, 85, 85);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <tabstops>
  <tabstop>Cancle_Button</tabstop>
  <tabstop>MOK_Button</tabstop>
  <tabstop>Ok_Button</tabstop>
  <tabstop>PwdLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>MCancel_Button</tabstop>
  <tabstop>Make_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
